CONTRIBUTING.md

```diff
@@ -441,10 +441,21 @@ The API reference is manually assembled in `doc/api/index.rst`.
 The *autodoc* sphinx extension will automatically create pages for each
 function/class/module listed there.
 
-You can reference classes, functions, and modules from anywhere (including docstrings)
-using <code>:func:\`package.module.function\`</code>,
-<code>:class:\`package.module.class\`</code>, or
-<code>:mod:\`package.module\`</code>.
+You can reference classes, functions, methods, and modules from anywhere 
+(including docstrings) using <code>:func:\`package.module.function\`</code>,
+<code>:class:\`package.module.class\`</code>, 
+<code>:meth:\`package.module.method\`</code>, or
+<code>:mod:\`package.module\`</code>. An example would be to use 
+<code>\`:meth:\`pygmt.Figure.grdview\`\`</code> to link 
+to :meth:`pygmt.Figure.grdview`. PyGMT documentation that is not a class, method, 
+or module can be linked with :doc:`Any Link Text </path/to/the/file>`
+And example would be <code>:doc:`Install instructions </install>`</code> links 
```

Comment:
```suggestion
For example, <code>:doc:`Install instructions </install>`</code> links 
```

Reply:
Also need a period in the line above, or to change "And" to "and an" and ignore the suggestion change.

Reply:
Grammar mistakes all around 😔

Thanks for the correction!